pkgs/applications/blockchains/wasabibackend/create_deps.sh

```diff
@@ -30,69 +43,56 @@ DEPS_TEMPLATE="
   sha256 = \"%s\";
 })"
 
+tmpdir="$(mktemp -d -p "$(pwd)")" # must be under source root
+trap 'rm -rf "$tmpdir"' EXIT
+
+HOME="$tmpdir" dotnet restore --packages "$tmpdir"/.nuget/packages \
+        --no-cache --force --runtime linux-x64 \
+        WalletWasabi.Backend/WalletWasabi.Backend.csproj >&2
+
+mapfile -t repos < <(
+    xmlstarlet sel -t -v 'configuration/packageSources/add/@value' -n NuGet.config "$tmpdir"/.nuget/NuGet/NuGet.Config |
+        while IFS= read index
+        do
+            curl --compressed -fsL "$index" | \
+                jq -r '.resources[] | select(."@type" == "PackageBaseAddress/3.0.0")."@id"'
+        done
+)
+
+echo $DEPS_HEADER >&6
+
+cd "$tmpdir/.nuget/packages"
+for package in *
+do
+    cd "$package"
+    for version in *
+    do
+        found=false
+        for repo in "${repos[@]}"
+        do
```

Comment:
Generally google style bash is preferred.

```
for x in y; do
```

Reply:
Yeah, I prefer that too; just copied that from somewhere else... Hope to fix all these things and use nuget-to-nix instead of all the different things done in those scripts.